### ISSUE TITLE
In tampermonkey menu with darker mode the icons are overlapping userscript switches

### ISSUE THREAD

**Issue by ftk789** [Tampermonkey/tampermonkey#1414]
(Please fill out the issue template with your details)

## Expected Behavior
Show icons of the scripts 

## Actual Behavior
Doesn't show any icons or shows glitched icons 

## Specifications

- Chrome Version 98.0.4758.82 (Official Build) (64-bit)

- OS: Windows 10

## Script
![image](https://user-images.githubusercontent.com/57598907/153223719-56750281-ffc6-4ce1-8cdc-fc89294e370a.png)

**Comment by derjanb** [comment]
Do you still see this after a Chrome restart? It looks like Chrome is still using CSS from the previous Tampermonkey version. If yes, then you should be able to fix the issue with this custom CSS setting.
Please use this value at Options > Appeareance -> Custom CSS to workaround the issue for the moment:

```CSS
 .actiontable .script .clickable > img {
display: none !important;
}
```

**Comment by ftk789** [comment]
![image](https://user-images.githubusercontent.com/57598907/153771918-389b95b5-5057-4f34-9cd6-f3551d755da5.png)
Fixed!, But the icons are gone now

**Comment by derjanb** [comment]
Should be fixed at 4.16.6157 (in review|[crx](https://tampermonkey.net/crx/tampermonkey_beta_current.crx)|[xpi](https://firefox.tampermonkey.net/firefox-current-beta.xpi))

Chrome/Edge users, please download the crx file linked above and drag and drop it to the extensions page `chrome://extensions`.
Firefox users please install the [BETA version](https://www.tampermonkey.net/index.php?browser=firefox) or check for BETA version updates at `about:addons`
